### ISSUE TITLE
Add feature and solve requirements.txt for windows

### ISSUE THREAD

**Issue by Farhanachyar** [Shaarav4795/ClippedAI#12]
Hello, nice to see your project. can you add some feature like auto subtitle for many language? and i think i can do like make a interface for some peopl e who dont understand to use cmd properly. and here a version for requirements.txt thats works for me on windows 11, Python 3.10 because python 3.11 so many trouble bcs conflicts dependencies. and i think i found some problem like, i trying to input a 16:9 video and the env aspec ratio is 9:16 but, the output still 16:9 without cropping. and there is 2 output that original one, and trimmed clip. but the trimmed clip still with 16:9 ratio. hope you fix it

absl-py==2.4.0
aiohappyeyeballs==2.6.1
aiohttp==3.13.3
aiosignal==1.4.0
alembic==1.18.4
annotated-doc==0.0.4
antlr4-python3-runtime==4.9.3
asteroid-filterbanks==0.4.0
attrs==26.1.0
av==17.0.0
certifi==2026.2.25
cffi==2.0.0
charset-normalizer==3.4.6
click==8.2.1
clipsai==0.2.1
colorama==0.4.6
colorlog==6.10.1
contourpy==1.3.2
ctranslate2==4.7.1
customtkinter==5.2.2
cycler==0.12.1
darkdetect==0.8.0
docopt==0.6.2
einops==0.8.2
facenet-pytorch==2.5.3
faster-whisper==1.2.1
filelock==3.25.2
flatbuffers==25.12.19
fonttools==4.62.1
frozenlist==1.8.0
fsspec==2026.2.0
greenlet==3.3.2
huggingface_hub==0.36.2
HyperPyYAML==1.2.3
idna==3.11
iniconfig==2.3.0
Jinja2==3.1.6
joblib==1.5.3
julius==0.2.7
kiwisolver==1.5.0
lightning==2.6.1
lightning-utilities==0.15.3
Mako==1.3.10
markdown-it-py==4.0.0
MarkupSafe==3.0.3
matplotlib==3.10.8
mdurl==0.1.2
mediapipe==0.10.33
mpmath==1.3.0
multidict==6.7.1
networkx==3.3
nltk==3.9.3
numpy==1.24.4
nvidia-ml-py==13.595.45
omegaconf==2.3.0
onnxruntime==1.23.2
opencv-contrib-python==4.11.0.86
opencv-python==4.11.0.86
optuna==4.8.0
packaging==26.0
pandas==2.3.3
pillow==12.1.1
platformdirs==4.9.4
pluggy==1.6.0
primePy==1.3
propcache==0.4.1
protobuf==4.25.8
psutil==7.2.2
pyannote.audio==3.1.1
pyannote.core==5.0.0
pyannote.database==5.1.3
pyannote.metrics==3.2.1
pyannote.pipeline==3.0.1
pycparser==3.0
Pygments==2.19.2
pynvml==13.0.1
pyparsing==3.3.2
pytest==9.0.2
python-dateutil==2.9.0.post0
python-dotenv==1.2.2
python-magic-bin==0.4.14
pytorch-lightning==2.6.1
pytorch-metric-learning==2.9.0
pytz==2026.1.post1
PyYAML==6.0.3
regex==2026.2.28
requests==2.32.5
rich==14.3.3
ruamel.yaml==0.18.17
ruamel.yaml.clib==0.2.15
safetensors==0.7.0
scenedetect==0.6.7.1
scikit-learn==1.3.2
scipy==1.10.1
semver==3.0.4
sentence-transformers==2.6.1
sentencepiece==0.2.1
shellingham==1.5.4
six==1.17.0
sortedcontainers==2.4.0
sounddevice==0.5.5
soundfile==0.13.1
speechbrain==1.0.3
SQLAlchemy==2.0.48
sympy==1.14.0
tabulate==0.10.0
tensorboardX==2.6.4
threadpoolctl==3.6.0
tokenizers==0.15.2
torch==2.1.2
torch-audiomentations==0.12.0
torch_pitch_shift==1.2.5
torchaudio==2.1.2
torchmetrics==1.9.0
torchvision==0.16.2
tqdm==4.67.3
transformers==4.36.2
typer==0.24.1
typing_extensions==4.15.0
tzdata==2025.3
urllib3==2.6.3
whisperx@git+https://github.com/m-bain/whisperx.git
yarl==1.23.0
